src/core/runtime/NOELLE_APIs.c

```diff
@@ -6,11 +6,20 @@ typedef struct {
 } DispatcherInfo;
 
 extern DispatcherInfo NOELLE_DOALLDispatcher(
-    void (*parallelizedLoop)(void *, int64_t, int64_t, int64_t),
+    void (*parallelizedLoop)(void *, int64_t, int64_t, int64_t, void *),
     void *env,
     int64_t maxNumberOfCores,
     int64_t chunkSize);
 
+extern int NOELLE_paraPrintfOneInt(int64_t coreID,
```

Comment:
I would consider making the name of this function more self-explanatory

Reply:
Agreed